config/hardware/axis/voron_switchwire_mk52.cfg

```diff
@@ -0,0 +1,18 @@
+# NOTE: Safe Printing Area of a Prusa Mk52 bed is
+# 250mm x 210mm at its most conservative, however
+# the endstop for a Voron Switchwire is located at
+# max Y (aka 240mm) in its stock configuration.
+
+[stepper_x]
+position_min: 0
+position_endstop: 250
+position_max: 250
+
+[stepper_y]
+position_min: 0
+position_endstop: 240
+position_max: 240
+
+[stepper_z]
+position_max: 235
+position_min: -3.0
```

Comment:
would it be possible to rename it a bit more generic with just adding the bedsize? like the "default_<ZZZ>mm.cfg" or "default_<XXX>mmX<YYY>mm.cfg"

Reply:
We could do that.  I think that the big issue that needs to be solved is that the travel for the SW is from -10,240 on Y and the silkscreen print on the Prusa Mk52 bed is shown as 250 x & 210 y (there are a pair of locating pegs that restrict the safe y print area at 0 < Z < 10 to this value).

So do we put 250x240 which is the physical size of the bed? Or 250x210 which is the safe print area?